src/subgraph/attributes.ts

```diff
@@ -47,7 +47,7 @@ export const FAIR_PRICE_SALE_FIELDS = gql`
     status
     startDate
     endDate
-    tokenAmount
+    tokensForSale
```

Comment:
`sellAmount`

Reply:
I'm looking at the aqua-xdai-dev subgraph and there is no parameter available for sellAmount in fairsale?